### ISSUE TITLE
Generating dart grpc client

### ISSUE THREAD

**Issue by gedw99** [uw-labs/proximo#75]
This project is really great. I have been thinking about a similar thing for ages 

I was having a look at the go-client. It looks like there is no reason why we can't codegen other language clients from the protobuffers, apart from a little housekeeping in the generated code. 

Are you open to other languages being done. We don't have to commit the generated code if you don't want.

**Comment by mjgarton** [comment]
Yes, we're open to more languages. Although we tend not to keep the client libraries in the main repo. Even the recommended go client is separate now (and is part of https://github.com/uw-labs/substrate )

**Comment by gedw99** [comment]
hey @mjgarton that's great to hear.

Yep will use substrate.
Will work it up in a separate repo for now wit a Makefile.

There is a nice simple chat demo over here i can try to get working.
https://github.com/amsokol/flutter-grpc-tutorial/tree/master/flutter_client/lib/api

The protobuf has no dependencies and works fine with BloomRPC GUI.

If anyone has time to lay out the steps go fo it.  i will just work through it and see how i go.
